.circleci/continue_config.yml

```diff
@@ -5073,3 +5145,63 @@ workflows:
         docker_image: "datadog/dd-trace-ci:php-8.3_centos-7"
         triplet: "aarch64-unknown-linux-gnu"
         resource_class: "arm.medium"
+
+  loader_tests:
+    when: << pipeline.parameters.loader >>
+    jobs:
+      - "compile_loader":
+          name: "Compile Linux"
+          docker_image: "datadog/dd-trace-ci:php-8.3_buster"
+          dd_library_package_url: https://github.com/DataDog/dd-trace-php/releases/download/0.99.1/dd-library-php-0.99.1-x86_64-linux-gnu.tar.gz
+
+      - "compile_loader":
+          name: "Compile Alpine"
+          docker_image: "datadog/dd-trace-ci:php-8.0_alpine"
+          dd_library_package_url: https://github.com/DataDog/dd-trace-php/releases/download/0.99.1/dd-library-php-0.99.1-x86_64-linux-musl.tar.gz
+
+      - "test_loader":
+          requires:
+            - "Compile Linux"
+          name: "[Linux] Loader PHP << matrix.php_major_minor >>-<< matrix.switch_php_version >>"
+          docker_image: "datadog/dd-trace-ci:php-<< matrix.php_major_minor >>_buster"
+          matrix:
+            parameters:
+              php_major_minor:
+#                - "5.6"
```

Comment:
We still need a test that, on 5.6, that the loader will disable itself. Without doing anything or emitting warnings. (Change the url to 1.0.0 to test without PHP 5 binaries included).